mapclassify/classifiers.py

```diff
@@ -809,6 +840,93 @@ def find_bin(self, x):
             right[right == len(self.bins)] = len(self.bins) - 1
         return right
 
+    def plot(
+        self,
+        gdf,
+        edge_color="lightgrey",
+        title=None,
+        legend=False,
+        cmap="YlGnBu",
+        axis_on=True,
+        legend_kwds={"loc": "lower right"},
+        file_name=None,
+        dpi=600,
+        ax=None,
+    ):
+        """
+        Plot mapclassiifer
+        NOTE: Requires matplotlib, and implicitly requires geopandas
+        dataframe as input.
+
+        Parameters
+        ---------
+        gdf         : geopandas geodataframe
+                      Contains the geometry column for the choropleth map
+        edge_color  : string
+                      matplotlib color string to use for polygon edges
```

Comment:
> The other thought I had was whether it makes sense to plug in here `vba` functionality that allows the user to "dim" observations by an ancillary variable that is passed. This would get us into full Swiss Army Knife for choroplething. But I'm not sure if this is the place as this is tied to a single classifier for the main variable. Maybe better to get this in and then write a simple dispatcher that checks whether the user wants `vba` or not and, depending, either sends it here or `choropleth_vba`? That method would probably fit better in `splot` I think (@slumnitz?).



> The other thought I had was whether it makes sense to plug in here `vba` functionality that allows the user to "dim" observations by an ancillary variable that is passed. This would get us into full Swiss Army Knife for choroplething. But I'm not sure if this is the place as this is tied to a single classifier for the main variable. Maybe better to get this in and then write a simple dispatcher that checks whether the user wants `vba` or not and, depending, either sends it here or `choropleth_vba`? That method would probably fit better in `splot` I think (@slumnitz?).

I agree that `splot` is a better home for such functionality.